modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1324,13 +1327,39 @@ CV__DNN_INLINE_NS_BEGIN
           */
          CV_WRAP ClassificationModel(const Net& network);
 
+         /**
+          * @brief The default for SoftMax option is false.
+          * If this option is true, SoftMax is applied after forward inference within the classify() function
+          * to convert the confidences range to [0.0-1.0].
+          * This function allows you to toggle this behavior.
+          * Please turn true when not contain SoftMax layer in model.
+          * @param[in] softmax Set apply/not-apply SoftMax within the classify() function.
+          */
+         CV_WRAP ClassificationModel& setSoftMax(bool apply);
```

Comment:
Usually it is written as `Softmax`/`softmax` (`max` is started from lowercase): 

What is about these names?
- `enableSoftmaxPostProcessing` / `isEnabledSoftmaxPostProcessing`
- `applySoftmaxPostProcessing` (? without getter)

Reply:
Overall, I have unified to `Softmax`. Thanks,

Reply:
It may be a bit redundant, I habe renamed set/get functions.
`setEnableSoftmaxPostProcessing` / `getEnableSoftmaxPostProcessing`